### PR TITLE
Added env.example file in the backend folder #66

### PR DESCRIPTION
**Pull request by coder12git**
Fixes: #66
Description: Added env. example and CODE_OF_CONDUCT.md file.
Attach Screenshots (if needed)

### CODE REVIEW

**Comment by J0SAL** [comment]
@Abhijit25Mishra please review it

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+This Code of Conduct applies within all community spaces, and also applies when
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at [Priyankar Pal](https://twitter.com/Priyankarpal.)
```
Comment (Abhijit25Mishra) [inline CODE_OF_CONDUCT.md:62]: change this to @J0SAL  github id

**Review by Abhijit25Mishra — CHANGES_REQUESTED** [review]
- ~~Improve PR title~~ Title seems fine.
- Update your PR template like this:
>Fixes: #99
>Description:
>Attach Screenshots (if needed)


also we are currently not looking for documentation changes and this will probably get rejected by @J0SAL .

**Comment by coder12git** [comment]
>

**Comment by coder12git** [comment]
@J0SAL I closed this pull request by mistake. But I had done all the changes as said by reviewers. Please see it and let me know if there is any problem.

**Pull request closed without merging**